content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -1,389 +1,316 @@
-    service.name: MyService
-    service.instance.id: 2c7ca153-e460-4643-b550-7c08487a4c0c
+app.Run();
 ```
 
-</details>
-
-### Manual Instrumentation
+In the `Properties` subdirectory, replace the content of `launchSettings.json`
+with the following:
 
-Next, add [tracing](/docs/concepts/signals/traces/) via the `System.Diagnostics`
-API.
-
-Paste the following code into your `HomeController`'s `Index` action:
-
-```csharp
-public IActionResult Index()
+```json
 {
-    // Track work inside of the request
-    using var activity = DiagnosticsConfig.ActivitySource.StartActivity("SayHello");
-    activity?.SetTag("foo", 1);
-    activity?.SetTag("bar", "Hello, World!");
-    activity?.SetTag("baz", new int[] { 1, 2, 3 });
-
-    return View();
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:8080",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
 }
 ```
 
-When you run the app and navigate to the `/` route, you'll see output about
-[spans](/docs/concepts/signals/traces/#spans) similar to the following:
+Build and run the application with the following command, then open
+<http://localhost:8080/rolldice> in your web browser to ensure it is working.
 
-<details>
-<summary>View example output</summary>
-
-```text
-Activity.TraceId:            47d25efc8b5e9184ce57e692f5f65465
-Activity.SpanId:             bb864adcf4592f54
-Activity.TraceFlags:         Recorded
-Activity.ParentSpanId:       acbff23f5ad721ff
-Activity.ActivitySourceName: MyService
-Activity.DisplayName:        SayHello
-Activity.Kind:               Internal
-Activity.StartTime:          2023-02-21T12:27:41.9596458Z
-Activity.Duration:           00:00:00.0005683
-Activity.Tags:
-    foo: 1
-    bar: Hello, World!
-    baz: [1,2,3]
-Resource associated with Activity:
-    service.name: MyService
-    service.instance.id: 2b07a9ca-29c4-4e01-b0ed-929184b32192
+```sh
+dotnet build
+dotnet run
 ```
 
-</details>
+## Instrumentation
 
-You'll notice the `Activity` objects from ASP.NET Core alongside the `Activity`
-we created manually in our controller action.
+Next, you'll use a [OpenTelemetry .NET Automatic Instrumentation](../automatic)
+to instrument the application at launch time. While you can [configure .NET
+Automatic Instrumentation][] in a number of ways, the steps below use Unix-shell
+or PowerShell scripts.
 
-### Metrics
+> **Note**: PowerShell commands require elevated (administrator) privileges.
 
-Next we'll add the ASP.NET Core automatically generated metrics to the app.
+1. Download installation scripts from [Releases][] of the
+   `opentelemetry-dotnet-instrumentation` repository:
 
-```csharp
-using OpenTelemetry.Resources;
-using OpenTelemetry.Trace;
-using OpenTelemetry.Metrics;
+   {{< tabpane text=true >}} {{% tab Unix-shell %}}
```

Comment:
What are the `tabpane` and `tab` added for? They are plainly rendered as text in the rich diff view.

Reply:
It's docs syntax to support tabbed content, see here: https://deploy-preview-3167--opentelemetry.netlify.app/docs/instrumentation/net/getting-started/#instrumentation